invenio_rdm_records/collections/service.py

```diff
@@ -131,3 +162,43 @@ def read_logo(self, identity, slug):
         if _exists:
             return url_for("static", filename=logo_path)
         raise LogoNotFoundError()
+
+    def read_many(self, identity, ids_=None, depth=2):
```

Comment:
I think it would be better to remove the default. It does not make much sense to have a `read_many` method without passing any id.
```suggestion
    def read_many(self, identity, ids_, depth=2):
```